### PR TITLE
v3.0.4

### PR DESCRIPTION
**Pull request by digitalartcrew**
* Use `triniti/schemas` v3.0.4
* Use `gdbots/schemas` v3.0.4

### CODE REVIEW

**Comment by digitalartcrew** [comment]
Don't merge until deployed: [Triniti Schema](https://github.com/triniti/schemas/pull/106)

**Comment by efjacobson** [comment]
@digitalartcrew need to compile them after adding the xml. also dont forget the event

File: composer.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   "require": {
     "php": ">=8.1",
```
Comment (efjacobson) [inline composer.json]: @digitalartcrew bump to >=8.3

File: composer.json
Comment (efjacobson) [inline composer.json:1]: @digitalartcrew lets change the semver for everything except pbjc to be `^N` eg

`"gdbots/schemas": "3.0.3"` -> `"gdbots/schemas": "^3",`

File: package.json
Comment (efjacobson) [inline package.json:1]: @digitalartcrew same `^N` changes here please

**Review by efjacobson — CHANGES_REQUESTED** [review]
@digitalartcrew bump the version in .nvmrc to at least v20.10.0

looking good overall though, just a few other minor things below

File: composer.json
Comment (efjacobson) [reply composer.json:1]: @digitalartcrew do it for all of them (except pbjc)

File: package.json
Comment (efjacobson) [reply package.json:1]: @digitalartcrew do it for all of them (including the dev dependencies)

**Review by efjacobson — APPROVED** [review]

**Pull request merged**